graviti/dataframe/column/series.py

```diff
@@ -109,6 +128,10 @@ def __repr__(self) -> str:
     def _get_repr_indices(self) -> Iterable[int]:
         return islice(range(len(self)), MAX_REPR_ROWS)
 
+    @staticmethod
+    def _from_pyarrow(array: pa.Array, schema: pt.PortexType) -> Any:
```

Comment:
why `_from_pyarrow` is `staticmethod` rather than `classmethod`, and why the return type is `Any`?